chain/types/index.go

```diff
@@ -0,0 +1,11 @@
+package types
+
+type IndexValidation struct {
+	TipsetKey string
```

Comment:
```suggestion
	TipSetKey TipSetKey
```

let's not do the string form, it's not very useful and the jsonrpc will convert the tipsetkey to an array of cids nicely enough